src/Markdowser/Utilities/CacheService.cs

```diff
@@ -0,0 +1,30 @@
+using Markdowser.ViewModels;
+using System.Collections.Generic;
+
+namespace Markdowser.Utilities
+{
+    namespace Markdowser.Utilities
+    {
+        public class CacheService
+        {
+            private Dictionary<string, ContentViewModelBase> cache = new Dictionary<string, ContentViewModelBase>();
+
+            public ContentViewModelBase Get(string url)
```

Comment:
Make return value nullable
```suggestion
            public ContentViewModelBase? Get(string url)
```